### PR TITLE
ONB-27: Mobile onboarding services

### PR DESCRIPTION
**Pull request by m4dz**
This PR enables responsive view of services styles in onboarding

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "preset name": "name",
     "step password empty": "Password's value is missing.",
     "step password too weak": "Your password is too weak. You should add an uppercase, a number, a special character or a longer password.",
-    "step password title": "Step 3: Choose your password",
```
Comment (gregorylegarec) [inline src/locales/en.json:38]: I did not see this point in the Trello card but I trust you. :)

**Review by gregorylegarec — APPROVED** [review]
Looks Good to @gregorylegarec

**Review by CPatchane — APPROVED** [review]
Seems good to me 👍

File: src/locales/en.json (hunk 1 above)
Comment (m4dz) [reply src/locales/en.json:38]: Yup, it's related to https://trello.com/c/oFbhV3ET/410-1-cc-7-amélioration-suite-aux-interviews-utilisateurs-locale, I took it to let @CPatchane go an other tasks :wink:

**Pull request merged**